crates/db_schema/src/impls/comment.rs

```diff
@@ -27,6 +28,59 @@ impl Comment {
       .get_result::<Self>(conn)
   }
 
+  // The child_count query for reference:
+  // select c.id, c.path, count(c2.id) as child_count from comment c
+  // left join comment c2 on c2.path <@ c.path and c2.path != c.path
+  // group by c.id
+
+  /// You must run this after every comment insert
```

Comment:
Sceptical about this. If its mandatory to call this method, it should be done automatically from Comment::create, Comment::upsert etc. Maybe by adding parent_path to CommentForm, or as method param.

Reply:
I'll see if I can move this stuff there.

Reply:
Actually I can't, because create is part of the Crud trait that can only take in a commentform, but this needs an optional parent_path.

Reply:
True, but i would just mark that method as `unimplemented!()`, and add a separate create method in the impl. Not that elegant but better to avoid bugs.

Reply:
Ah okay, I'll give it a try.